src/maptrait.rs

```diff
@@ -3,9 +3,14 @@ use std::str::FromStr;
 use std::hash::Hash; 
 use std::collections::HashMap; 
 
+
+///Trait for custom implementation of formatting 
+///Specifies two methods that can be used for formatting 
+///Any custom object implementing Map can be used for the formatting variable stores 
 pub trait Map<K , V> where K: Hash + Eq + FromStr, V: DisplayStr{
     
-    ///Prioritized first 
+    /// Prioritized first 
```

Comment:
What does this comment mean? Please remove it.

Reply:
There are two functions in the `Map` trait, `get `and `get_owned`. get is prioritized first. Only if `get `returns `None`, the `get_owned `method call will be performed. That's why the comment is there.